src/login/styles/_buttons.scss

```diff
@@ -208,6 +209,10 @@ button.btn-link.nobutton.verify-status-label {
   }
 }
 
+.show-hide-button {
+  margin: 0;
+}
+
```

Comment:
It looks like `.show-hide-button` appears multiple times in the file, if that's the case I would recommend that they'd be combined

Reply:
The reason "show-hide-button" is divided in two, is that the styling is common with the other buttons
The only application specific styling for "show-hide-button" is the "margin"
If I where to combine all the styling for "show-hid-button" that would be a duplication of the code.